[2020-11-24 11:41:21,660] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: treino-02.get-data 2020-11-24T11:00:00+00:00 [queued]>
[2020-11-24 11:41:21,666] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: treino-02.get-data 2020-11-24T11:00:00+00:00 [queued]>
[2020-11-24 11:41:21,666] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2020-11-24 11:41:21,666] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2020-11-24 11:41:21,666] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2020-11-24 11:41:21,673] {taskinstance.py:901} INFO - Executing <Task(BashOperator): get-data> on 2020-11-24T11:00:00+00:00
[2020-11-24 11:41:21,675] {standard_task_runner.py:54} INFO - Started process 76861 to run task
[2020-11-24 11:41:21,692] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'treino-02', 'get-data', '2020-11-24T11:00:00+00:00', '--job_id', '28', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/treino02.py', '--cfg_path', '/tmp/tmp1fhd3h1n']
[2020-11-24 11:41:21,692] {standard_task_runner.py:78} INFO - Job 28: Subtask get-data
[2020-11-24 11:41:21,759] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: treino-02.get-data 2020-11-24T11:00:00+00:00 [running]> gaming
[2020-11-24 11:41:21,768] {bash_operator.py:113} INFO - Tmp dir root location: 
 /tmp
[2020-11-24 11:41:21,769] {bash_operator.py:134} INFO - Temporary script location: /tmp/airflowtmpi3km0i1j/get-datac4sngty9
[2020-11-24 11:41:21,769] {bash_operator.py:146} INFO - Running command: curl https://raw.githubusercontent.com/A3Data/hermione/master/hermione/file_text/train.csv -o ~/raw/train.csv
[2020-11-24 11:41:21,773] {bash_operator.py:153} INFO - Output:
[2020-11-24 11:41:21,800] {bash_operator.py:157} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2020-11-24 11:41:21,800] {bash_operator.py:157} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2020-11-24 11:41:22,485] {bash_operator.py:157} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to create the file /home/thomaz/raw/train.csv: No such file or
[2020-11-24 11:41:22,486] {bash_operator.py:157} INFO - Warning: directory
[2020-11-24 11:41:22,486] {bash_operator.py:157} INFO -   2 60301    2  1371    0     0   1998      0  0:00:30 --:--:--  0:00:30  1995
[2020-11-24 11:41:22,488] {bash_operator.py:157} INFO - curl: (23) Failure writing output to destination
[2020-11-24 11:41:22,490] {bash_operator.py:159} INFO - Command exited with return code 23
[2020-11-24 11:41:22,498] {taskinstance.py:1150} ERROR - Bash command failed
Traceback (most recent call last):
  File "/home/thomaz/anaconda3/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/thomaz/anaconda3/lib/python3.8/site-packages/airflow/operators/bash_operator.py", line 165, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2020-11-24 11:41:22,499] {taskinstance.py:1187} INFO - Marking task as UP_FOR_RETRY. dag_id=treino-02, task_id=get-data, execution_date=20201124T110000, start_date=20201124T144121, end_date=20201124T144122
[2020-11-24 11:41:26,650] {local_task_job.py:102} INFO - Task exited with return code 1
